### ISSUE TITLE
Using tensorboard with tf.keras 1.14

### ISSUE THREAD

**Issue by zaher88abd** [tensorflow/tensorboard#2584]
I am using Tensorboard with Tf.keras, the issue I have is the Tensorboard is not reading the data new data, I have to stop and start the Tensorboard to see the new loss values. 
This the code for creating the Tensorboard callback:
```
callbacks = [k.callbacks.TensorBoard("./log/" + results.model_name + "/",
                                             update_freq='batch'), model_check_point]
``` 
Reagarding the versions:
tensorflow-gpu                1.14.0
tensorboard                   1.14.0

**Comment by wchargin** [comment]
Hi @zaher88abd—could you try adding `profile_batch=0` to your Keras
callback? You may be running into #2084.

**Comment by zaher88abd** [comment]
Thank you @wangpengmit, it's working now.

**Comment by wchargin** [comment]
Great! Glad to hear it.

**Comment by wchargin** [comment]
Duplicate of #2084
